shacl12-ui/index.html

```diff
@@ -1513,6 +1513,142 @@ <h4>Other Complex Paths</h4>
         </section>
     </section>
 
+    <section id="property-roles">
+      <h2>Property Roles</h2>
+
+      <p>
+        RDF resources commonly use properties to specify labels and descriptions, which user-interface engines depend on to render structured content.
```

Comment:
```suggestion
        RDF resources commonly use properties to specify labels and descriptions, and other information needed for rendering, which user-interface engines depend on to render structured content.
```

Reply:
I have icons and images in mind, but I wanted to keep the text generic in case we find other use cases.

Reply:
Small tweak to @bergos' tweak.
```suggestion
        RDF resources commonly use properties to specify labels, descriptions, and other information needed for rendering, which user-interface engines depend on to render structured content.
```